flatgeobuf/flatgeobuf.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>make a note box</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I had assumed the special note box syntax didn't work from inside notebooks, but I'll move to using that.